### ISSUE TITLE
Medusa Development Resources: Issue in CORS Errors - Medusa Development Resources

### ISSUE THREAD

**Issue by lindyhan** [medusajs/medusa#9791]
### What Medusa version and documentation are you using?

None

### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/medusajs/medusa/issues

### Issue Summary

there is cors error at 9000. the default code upon cloning is already the same code as in the docs and it does not work

### How can this issue be resolved?

1.
2.
3.
...

### Are you interested in working on this issue?

- [ ] I would like to fix this issue

**Comment by sradevski** [comment]
@lindyhan can you please provide a bit more information on the steps you did, how you get the error and so on?

**Comment by lindyhan** [comment]
Hi,

I followed everything in the readme page, until the localhost9000 admin. i
am able to display the login box but after i click "continue to email", i
get this in the console logs

Access to fetch at '
https://sofa-society-starter.medusajs.app/auth/user/emailpass' from origin '
http://localhost:9000' has been blocked by CORS policy: Response to
preflight request doesn't pass access control check: No
'Access-Control-Allow-Origin' header is present on the requested resource.
If an opaque response serves your needs, set the request's mode to
'no-cors' to fetch the resource with CORS disabled.
chunk-7CT6GXPZ.js?v=7b95917f:8948


       POST https://sofa-society-starter.medusajs.app/auth/user/emailpass
net::ERR_FAILED
(anonymous) @ chunk-7CT6GXPZ.js?v=7b95917f:8948
(anonymous) @ chunk-7CT6GXPZ.js?v=7b95917f:8738
__awaiter43 @ chunk-7CT6GXPZ.js?v=7b95917f:8720
(anonymous) @ chunk-7CT6GXPZ.js?v=7b95917f:8926
fetch @ chunk-7CT6GXPZ.js?v=7b95917f:8876
(anonymous) @ chunk-7CT6GXPZ.js?v=7b95917f:8553
(anonymous) @ chunk-7CT6GXPZ.js?v=7b95917f:8539
__awaiter42 @ chunk-7CT6GXPZ.js?v=7b95917f:8521
Auth.login @ chunk-7CT6GXPZ.js?v=7b95917f:8552
mutationFn @ chunk-EKJ7BVDF.js?v=7b95917f:11
fn @ chunk-Z5Y2OGOT.js?v=7b95917f:1189
run @ chunk-Z5Y2OGOT.js?v=7b95917f:494
start @ chunk-Z5Y2OGOT.js?v=7b95917f:536
execute @ chunk-Z5Y2OGOT.js?v=7b95917f:1225
await in execute
mutate @ chunk-Z5Y2OGOT.js?v=7b95917f:2619
(anonymous) @ login-7GLOPTH7-4ZDDO2DN.js?v=7b95917f:114
(anonymous) @ chunk-M3SY7T4C.js?v=7b95917f:1603
await in (anonymous)
callCallback2 @ chunk-C4VVQZP3.js?v=7b95917f:3674
invokeGuardedCallbackDev @ chunk-C4VVQZP3.js?v=7b95917f:3699
invokeGuardedCallback @ chunk-C4VVQZP3.js?v=7b95917f:3733
invokeGuardedCallbackAndCatchFirstError @ chunk-C4VVQZP3.js?v=7b95917f:3736
executeDispatch @ chunk-C4VVQZP3.js?v=7b95917f:7014
processDispatchQueueItemsInOrder @ chunk-C4VVQZP3.js?v=7b95917f:7034
processDispatchQueue @ chunk-C4VVQZP3.js?v=7b95917f:7043
dispatchEventsForPlugins @ chunk-C4VVQZP3.js?v=7b95917f:7051
(anonymous) @ chunk-C4VVQZP3.js?v=7b95917f:7174
batchedUpdates$1 @ chunk-C4VVQZP3.js?v=7b95917f:18913
batchedUpdates @ chunk-C4VVQZP3.js?v=7b95917f:3579
dispatchEventForPluginEventSystem @ chunk-C4VVQZP3.js?v=7b95917f:7173
dispatchEventWithEnableCapturePhaseSelectiveHydrationWithoutDiscreteEventReplay
@ chunk-C4VVQZP3.js?v=7b95917f:5478
dispatchEvent @ chunk-C4VVQZP3.js?v=7b95917f:5472
dispatchDiscreteEvent @ chunk-C4VVQZP3.js?v=7b95917f:5449
Show 32 more frames
Show less
chunk-7CT6GXPZ.js?v=7b95917f:8948




       Uncaught (in promise) TypeError: Failed to fetch
    at Client.<anonymous> (chunk-7CT6GXPZ.js?v=7b95917f:8948:20)
    at Generator.next (<anonymous>)
    at chunk-7CT6GXPZ.js?v=7b95917f:8738:67
    at new Promise (<anonymous>)
    at __awaiter43 (chunk-7CT6GXPZ.js?v=7b95917f:8720:10)
    at Client.fetch_ (chunk-7CT6GXPZ.js?v=7b95917f:8926:29)
    at Client.fetch (chunk-7CT6GXPZ.js?v=7b95917f:8876:17)
    at Auth.<anonymous> (chunk-7CT6GXPZ.js?v=7b95917f:8553:53)
    at Generator.next (<anonymous>)
    at chunk-7CT6GXPZ.js?v=7b95917f:8539:67


[image: image.png]

**Comment by lindyhan** [comment]
Hi, it's been resolved. Please ignore now. Thanks for your time!

**Comment by IcooN** [comment]
@lindyhan can I ask how did you resolved it? I have the same issue right now.

**Comment by lindyhan** [comment]
config.medusa.js has this line
`backendUrl: process.env.BACKEND_URL`

but it's not in the env template
so add it into your env
`BACKEND_URL=http://localhost:9000`

**Comment by IcooN** [comment]
@lindyhan thank you it's working :)

**Comment by lindyhan** [comment]
Cool! 👏🏻
